src/components/ClientsPage/introduction.js

```diff
@@ -0,0 +1,142 @@
+import { SB_NAVY } from "@colors"
+import ClientPageProjectImage from "content/clients/project.jpeg"
+import React from "react"
+import styled from "styled-components"
+import Body from "styles/components/Body"
+import { Header } from "styles/components/Header"
+import bulletPoint from "images/sandbox-bullet.svg"
+import PropTypes from "prop-types"
+
+const BlueBackgroundSection = styled.div`
+  background-color: ${SB_NAVY};
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 5em 2em 5em 2em;
+  margin: 0 auto;
+
+  @media (min-width: 1000px) {
+    min-height: 20vh;
+    max-width: 100vw;
+  }
+`
+
+const IntroductionContainer = styled.div`
+  max-width: 40%;
+  @media (max-width: 1100px) {
+    max-width: 700px;
+    width: 100%;
+  }
+`
+
+const ProjectContainer = styled.div`
+  max-width: 60%;
+  padding: 4em 4em 0em 4em;
+  text-align: right;
+  @media (max-width: 1400px) {
+    padding: 4em 0em 0em 4em;
+  }
+  @media (max-width: 1100px) {
+    max-width: 100%;
+    width: 100%;
+    padding: 4em 4em 0em 4em;
+  }
+
+  @media (max-width: 600px) {
+    max-width: 100%;
+    width: 100%;
+    padding: 4em 0em 0em 0em;
+  }
+`
+
+const ProjectImage = styled.img`
+  width: 90%;
+  height: auto;
+  @media (max-width: 1400px) {
+    width: 95%;
+  }
+  @media (max-width: 1100px) {
+    width: 100%;
+  }
+`
+const ProjectLabel = styled.p`
+  color: #fff;
+  font-size: 24px;
+  font-family: Brandon;
+  @media (max-width: 600px) {
+    font-size: 20px;
+  }
+`
+
+const WhitePageHeader = styled(Header)`
+  color: white;
+  font-size: 4em;
+  letter-spacing: 0.02em;
+`
+
+const BodyText = styled(Body)`
+  color: white;
+`
+
+const ListContainer = styled.div`
+  margin: 35px 0px 0px 0px;
+  padding: 0px;
+`
+
+const ListItemContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin: 20px 0px;
+`
+
+const ListItem = styled.p`
+  color: white;
+  font-size: 1.4em;
+  line-height: 1.1;
+  margin: 0px 0px 0px 20px;
+`
+
+const BulletPointContainer = styled.div`
+  width: 27px;
+`
+
+const ClientPageIntroduction = ({ reasonsToWorkWithSandbox }) => {
+  return (
+    <BlueBackgroundSection>
+      <IntroductionContainer>
```

Comment:
i think we may need to adjust the margins on the page or components, in general, I think the margins on the left and right could be increased so that the words aren't so close to the side of the screen.

going off of the figma, it looks like the "work with sandbox" could line up more with the logo in the navbar to give it some more space. I'm not sure what would be easiest if we can add a margin to the whole page or to the individual components.